apps/sms/views/shared/test/unit/activity_handler_test.js

```diff
@@ -943,6 +940,24 @@ suite('ActivityHandler', function() {
         );
       }).then(done,done);
     });
+
+    test('existing thread with specified body, navigate to composer',
```

Comment:
you're not wrong, but I'm not wrong either :)

from the unit test writer perspective, we don't know how the code works. So I think the tests as they are now reflect better the "black box" we're testing.

Using a `sinon.assert.notCalled` means we know what happens inside the code. In the end, we don't really care whether it's called or not, as long as we move to the right panel. What's important is our 2 cases: there is a known conversation, there is not a known conversation => what do we do ? Currently we do the same, that's right :) but the 2 cases are not less different.

What do you think ?